portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Oyin's Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+        }
```

Comment:
The spacing is a little off here

```suggestion
    body {
        font: 22px 'Quicksand', sans-serif;
        background-color: #a8d0e6;
    }
```

Reply:
I will make sure to keep it at 2 indents and fix the whole thing, thank you.

Reply:
Thanks :) one tiny nit - there should be a space after body: `body {`